projects/cdk/schematics/ng-update/steps/replace-styles.ts

```diff
@@ -0,0 +1,16 @@
+import {getActiveProject} from 'ng-morph';
+
+export function replaceStyles() {
+    getActiveProject()
+        ?.getSourceFiles('**/**.less')
+        .forEach(sourceFile => {
+            const fullText = sourceFile.getFullText();
+
+            sourceFile.replaceWithText(
+                fullText.replace(
+                    `@import '~@taiga-ui/core/styles/taiga-ui-global';`,
+                    `@import '~@taiga-ui/styles/taiga-ui-fonts';\n@import '~@taiga-ui/styles/taiga-ui-global';`,
```

Comment:
Fonts has to be in core because it's a part of the theme.

Reply:
fixed